### PR TITLE
app-editor/vim: Add v9.0.0260

### PR DESCRIPTION
**Pull request by GZGavinZhao**
Signed-off-by: Gavin Zhao <git@gzgz.dev>

### CODE REVIEW

File: app-editor/vim/stone.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+homepage    : https://github.com/vim/vim
+upstreams   :
+    - https://github.com/vim/vim/archive/refs/tags/v9.0.0260.tar.gz : 6cc50bc98b7273998fbcf4e3fd4334c9a9c1a0551ec8f99e7c92f8554ad7f596
+summary     : Vi IMproved, an advanced text editor
+description : |
+    Vim, which stands for Vi IMproved, is an open-source, multiplatform text editor extended from vi. It was first released by Bram Moolenaar in 1991. Since then, numerous features have been added to Vim, many of which are helpful in editing program source code.
+license     :
+    - Vim
+builddeps   :
+    - pkgconfig(ncursesw)
```
Comment (ReillyBrogan) [inline app-editor/vim/stone.yml:33]: Are there any deps that would be good to build this with but that are not packaged yet? If so you could put some `# TODO` comments here with what they are and what they enable. Eventually as the repo expands and those are brought in we can rebuild this package with them.

Comment (GZGavinZhao) [reply app-editor/vim/stone.yml:33]: Done. They are mainly for extra providers and Vim's GTK front-end.

Comment (GZGavinZhao) [reply app-editor/vim/stone.yml:33]: Welp, I basically just copied them from the other operating system's build file.

Comment (ermo) [reply app-editor/vim/stone.yml:33]: Hey there,

As this is a recipe monorepo with lots of other recipes, friendly reminder that the commit message needs to have version info when adding/updating packages for clarity of shortlog view.

I added this guidance to the README last night:

### Commit messages

Use a category/pkg: verb <foo> format with one commit per package change.

Example: `shells/fish: Add vX.Y` or `shells/fish: Update to vX.Y` or `shells/fish: Fix (...)`

Bump release for functional recipe changes only; do not bump release for no functional change commits (documentation typos, formatting).

No functional change commits use the [NFC] tag: `shells/fish: [NFC] Tweak comments.`

Comment (GZGavinZhao) [reply app-editor/vim/stone.yml:33]: Done.

**Comment by ermo** [comment]
LGTM, thanks!

**Pull request merged**